Documentation/user-guides/webhook.md

```diff
@@ -40,67 +48,181 @@ metadata:
   namespace: default
 ```
 
-The admission webhook's pod template should be modified to mount the secret as a
-volume and the container arguments should be modified to include the
-certificate and key:
+The recommended approach is to use [cert-manager](https://cert-manager.io/)
+which manages both the lifecycle of the TLS certificates and the integration
+with the Kubernetes API with respect to the webhook configuration (e.g.
+automatic injection of the CA bundle).
+
+While installing cert-manager is beyond the scope of this guide, below is an
+example of a `Certificate` object which triggers the creation of the
+`admission-webhook-certs` secret using a [SelfSigned
+issuer](https://cert-manager.io/docs/configuration/selfsigned/).
 
 ```yaml
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: prometheus-operator-admission-webhook
+  namespace: default
+spec:
+  dnsNames:
+    - prometheus-operator-admission-webhook.default.svc
```

Comment:
Do you think it's useful to add a small note mentioning that the `dnsName`  should follow the format `<name of the service exposing the webhook>.<namespace where the webhook is deployed>.svc`?

Reply:
sure